server.py

```diff
@@ -146,6 +143,38 @@ def dashboard():
     contests = Contest.select().order_by(Contest.start_time)
     return bottle.template("dashboard.html", contests=contests)
 
+@app.get("/stats")
+@login_required
+def statistics():
+    sub_history = []
+    sub_stats_correct = []
+    sub_stats_total = []
+    sub_history_temp = Submission.select(Contest.code, ContestProblems.question, Submission.time, Submission.is_correct)\
+        .join(ContestProblems, on=(Submission.contestProblem == ContestProblems.id)) \
+        .join(Session, on=(Submission.user == Session.user))\
```

Comment:
I think I see a problem. Try this out:

1. add 2 users
2. make some submissions for both of them
3. see stats and find your mistake